README.md

```diff
@@ -12,8 +12,14 @@ $ source setup.sh
 ```bash
 $ source setup.sh --windows
 ```
+Setup the below Environment variables for Trello API
 ```
-update trello_items.py with trello key and token.
+KEY
+TOKEN
+BOARD_ID
+TODO_LIST_ID
+DOING_LIST_ID
+DONE_LIST_ID
 ```
```

Comment:
Good job updating this, but your README should also mention how to run the tests and any prerequisites (geckodriver & Firefox).

You should add these with placeholder values to .env.template.

And minor point, but it would be good to use more specific env variable names that "KEY" and "TOKEN", e.g. "TRELLO_KEY"

Reply:
done